.github/workflows/breaking-change-review.yml

```diff
@@ -0,0 +1,135 @@
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number,
+              reviewers: reviewers
+            });
+
+      - name: Check QA approvals
+        if: steps.check_label.outputs.result == 'true'
+        id: check_approvals
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const { data: reviews } = await github.rest.pulls.listReviews({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number,
+            });
+            
+            const mobileQA = '${{ steps.set_reviewers.outputs.mobile_qa }}'.split(',');
+            const desktopQA = '${{ steps.set_reviewers.outputs.desktop_qa }}'.split(',');
+            
+            const mobileQAApproved = reviews.some(review => 
+              review.state === 'APPROVED' && mobileQA.includes(review.user.login)
+            );
+            const desktopQAApproved = reviews.some(review => 
+              review.state === 'APPROVED' && desktopQA.includes(review.user.login)
+            );
+            
+            console.log(`Mobile QA approved: ${mobileQAApproved}`);
+            console.log(`Desktop QA approved: ${desktopQAApproved}`);
+            
+            core.setOutput('mobile-qa-approved', mobileQAApproved);
+            core.setOutput('desktop-qa-approved', desktopQAApproved);
+            
+            return mobileQAApproved && desktopQAApproved;
+
+      - name: Block PR if conditions not met
+        if: steps.check_label.outputs.result == 'true' && steps.check_approvals.outputs.result != 'true'
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const mobileQAApproved = ${{ steps.check_approvals.outputs.mobile-qa-approved }};
+            const desktopQAApproved = ${{ steps.check_approvals.outputs.desktop-qa-approved }};
+            
+            let message = 'This PR has the breaking change label and requires approval from both mobile-qa and desktop-qa teams before it can be merged.\n\n';
+            
+            if (!mobileQAApproved && !desktopQAApproved) {
+              message += 'Both mobile-qa and desktop-qa teams have not approved this PR yet.';
+            } else if (!mobileQAApproved) {
+              message += 'The mobile-qa team has not approved this PR yet.';
+            } else if (!desktopQAApproved) {
+              message += 'The desktop-qa team has not approved this PR yet.';
+            }
+            
+            await github.rest.checks.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              name: 'QA Approval Check',
+              head_sha: context.payload.pull_request.head.sha,
+              status: 'completed',
+              conclusion: 'failure',
```

Comment:
@igor-sirotin : added a failing check as discussed, the github-actions bot will no longer request changes, It will just add a comment with an appropriate message.

Reply:
Does this create a separate status check? Maybe we could just use this check itself?
Something like this should be enough I think 🤔 
https://github.com/status-im/status-go/blob/a1c6d7f333942f15221b75a588292aae420872d8/.github/workflows/commit-check.yml#L100

The reason is not to bloat the checks list. It's already 12 😅 

And the comment can be implemented as "sticky comment", I did it in the `commit-check`. Can be a single comment in the PR that will be edited when reviews received/lost.